packages/holocron/docs/api/README.md

```diff
@@ -10,29 +10,72 @@ available on the server. We have organized them as such.
-  - [`registerModule`](#registermodule)
-  - [`getModule`](#getmodule)
-  - [`getModules`](#getmodules)
-  - [`getModuleMap`](#getmodulemap)
-  - [`setModuleMap`](#setmodulemap)
-- [Selectors](#selectors)
-  - [`isLoaded`](#isloaded)
-  - [`failedToLoad`](#failedToLoad)
-  - [`getLoadError`](#getLoadError)
-  - [`isLoading`](#isloading)
-  - [`getLoadingPromise`](#getloadingpromise)
-- [Advanced Functions](#advanced-functions)
-  - [`forceLoadModule`](#forceloadmodule)
+- [🎛 API](#-api)
+  - [Universal](#universal)
+    - [App-level Functions](#app-level-functions)
+      - [`createHolocronStore`](#createholocronstore)
+        - [Arguments](#arguments)
+        - [Usage](#usage)
+    - [Holocron Module Configuration](#holocron-module-configuration)
+      - [`Module.holocron`](#moduleholocron)
+        - [Properties](#properties)
+          - [options object](#options-object)
+      - [Usage](#usage-1)
+    - [Module-level Functions](#module-level-functions)
+      - [`RenderModule`](#rendermodule)
+        - [Props](#props)
+        - [Usage](#usage-2)
+      - [`composeModules`](#composemodules)
+        - [Arguments](#arguments-1)
+        - [Usage](#usage-3)
+      - [`loadModule`](#loadmodule)
+        - [Arguments](#arguments-2)
+        - [Usage](#usage-4)
+      - [`holocronModule (Deprecated)`](#holocronmodule-deprecated)
+        - [Arguments](#arguments-3)
+        - [Usage](#usage-5)
+    - [Module Registry](#module-registry)
+      - [`registerModule`](#registermodule)
+        - [Arguments](#arguments-4)
+      - [`getModule`](#getmodule)
+        - [Arguments](#arguments-5)
+        - [Usage](#usage-6)
+      - [`getModules`](#getmodules)
+        - [Arguments](#arguments-6)
+        - [Usage](#usage-7)
+      - [`getModuleMap`](#getmodulemap)
+        - [Arguments](#arguments-7)
+        - [Usage](#usage-8)
+      - [`setModuleMap`](#setmodulemap)
+        - [Arguments](#arguments-8)
+        - [Usage](#usage-9)
+    - [Selectors](#selectors)
+      - [`isLoaded`](#isloaded)
+        - [Arguments](#arguments-9)
+        - [Usage](#usage-10)
+      - [`failedToLoad`](#failedtoload)
+        - [Arguments](#arguments-10)
+        - [Usage](#usage-11)
+      - [`getLoadError`](#getloaderror)
+        - [Arguments](#arguments-11)
+        - [Usage](#usage-12)
+      - [`isLoading`](#isloading)
+        - [Arguments](#arguments-12)
+        - [Usage](#usage-13)
+      - [`getLoadingPromise`](#getloadingpromise)
+        - [Arguments](#arguments-13)
+        - [Usage](#usage-14)
+  - [Server](#server)
+      - [`updateModuleRegistry`](#updatemoduleregistry)
+        - [Arguments](#arguments-14)
+        - [Usage](#usage-15)
+      - [`areModuleEntriesEqual`](#aremoduleentriesequal)
+        - [Arguments](#arguments-15)
+        - [Usage](#usage-16)
+    - [Advanced Functions](#advanced-functions)
+      - [`forceLoadModule`](#forceloadmodule)
+        - [Arguments](#arguments-16)
+        - [Usage](#usage-17)
```

Comment:
could this chunk be reverted, or are we happy with it being here now its been generated?

Reply:
reverted